doc/api/stream.md

```diff
@@ -1445,9 +1445,9 @@ added: v8.0.0
 -->
 
 * `callback` {Function} Call this function (optionally with an error
-  argument) when you are done writing any remaining data.
+  argument) when finished writing any remaining data.
 
-Note: `_final()` **must not** be called directly.  It MAY be implemented
+*Note*: `_final()` **must not** be called directly. It MAY be implemented
```

Comment:
The uppercase variant was most likely derived from [RFC2119](https://tools.ietf.org/html/rfc2119), which is used in a lot of non-RFC documents. (I am not saying we SHOULD follow that.)

±0 on keeping it a `Note`.